cmd/util/util.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 The Kubernetes Authors.
```

Comment:
I'm not sure we should place a util library under the `cmd` hierarchy. Shouldn't `util.go` be under `pkg`